cmd/server/main.go

```diff
@@ -28,15 +28,20 @@ func main() {
 			Usage: "ocid pod sandbox dir",
 		},
 		cli.StringFlag{
-			Name:  "runtime",
-			Value: "/usr/bin/runc",
-			Usage: "OCI runtime path",
-		},
-		cli.StringFlag{
 			Name:  "containerdir",
 			Value: "/var/lib/ocid/containers",
 			Usage: "ocid container dir",
 		},
+		cli.StringFlag{
+			Name:  "socket",
+			Value: unixDomainSocket,
+			Usage: "path to ocid socket",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Value: "/usr/bin/runc",
```

Comment:
Doesn't matter then. Besides, I'll move `runc` back to `/usr/local/bin` once we get rootless containers merged (the `sbin` stuff was just because it's technically a root-only tool at the moment).